front/package.json

```diff
@@ -41,8 +50,10 @@
     "cmdk": "1.0.0",
     "lucide-react": "^0.453.0",
     "next": "14.2.15",
+    "next-themes": "^0.4.1",
```

Comment:
what is this for?

Reply:
Next Themes has been added by the toaster component, to handle the theme correctly in the toaster ig

Reply:
Update: only used in the shadcn one so I could remove without any issue